server.hcl

```diff
@@ -156,17 +155,6 @@ target "server-tensorflow" {
     }
 }
 
-target "server-ui" {
-    inherits = [ "server-context" ]
-    tags = [
-        "${REPO_PREFIX}${SERVER_PREFIX}-ui:${TAG}",
-        equal("latest", TAG) ? "${REPO_PREFIX}${SERVER_PREFIX}-ui:${DEEPHAVEN_VERSION}" : ""
```

Comment:
We can continue producing `server-ui` images by just tagging the `server` images with these tags in the server target; I think we should prefer to do this for at least one release, at which point we can consider removing them.